03-metadata-formats-and-files.Rmd

```diff
@@ -104,110 +89,129 @@ plot(DSM_HARV,
 
 **Spatial Information**
 
 * **Spatial Extent:** What area does this dataset cover? 
-* **Coordinate reference system:** what spatial projection / coordinate reference
+* **Coordinate reference system:** What spatial projection / coordinate reference
 system are used to store the data? Will it line up with other data? 
 * **Resolution:** The data appears to be in **raster** format. This means it is
-composed of pixels. What area on the ground does each pixel cover - ie What is 
+composed of pixels. What area on the ground does each pixel cover - i.e. What is 
 its spatial resolution?
 
 **Data Collection / Processing Methods**
 
+* **When was the data collected?** Is it recent or historical?
 * **How was this data generated?** Is this an output from a model, is it an image
-from a remote sensing instrument such as a satellite (e.g. landsat) or collected
-from an airplane? How were the data colected?
-* **Units:** We can see a scale bar of values to the right of the data however, 
-what units / metric does this represent? Temperature? Elevation? Precipitation?
+from a remote sensing instrument such as a satellite (e.g. Landsat) or collected
+from an airplane? How were the data collected?
+* **Units:** We can see a scale bar of values to the right of the data, however, 
+what metric and in what units does this represent? Temperature? Elevation? Precipitation?
 * **How were the data processed?**
 
 **Contact Information**
-* Who created this data? Who do we contact if we have questions?
 
-When we are given a dataset, or when we download it online, we do not know anything
-about it without proper documentation. This documentation is called **metadata** - 
-data about the data.
+* **Who created this data?** 
+* **Who do we contact:** We might need permission to use it, have questions
+about the data, or need more information to give correct attribution.
+
+When we are given a dataset, or when we download it online, we do not know
+anything about it without proper documentation. This documentation is called 
+**metadata** - data about the data.
 
-## What is Metadata?
-Metadata is structured information that describes a dataset. It includes a suite of
-information about the data including:
+## What are Metadata?
+Metadata are structured information that describes a dataset. Metadata include 
+a suite of information about the data including:
 
-* Contact information
-* Spatial Attributes including: extent, coordinate reference system, resolution
-* Data collection & processing methods
-* and more.
+* Contact information,
+* Spatial Attributes including: extent, coordinate reference system, resolution,
+* Data collection & processing methods,
+* and much more.
 
-Without sufficient documentation, it is difficult for us to work with 
-external data - data that we did not collect ourselves.
+Without sufficient documentation, it is difficult for us to work with external 
+data - data that we did not collect ourselves.
 
 ### Why Are Metadata Needed?
```

Comment:
I was very confused by the text that was in here (old lines 166-170). Plus it brought in the idea of machine ready files BEFORE we introduce that concept in the next section.  
Not sure I captured the full idea that you were going for with my changes.  Plus with a title of Why Needed? this section could be more extensive.